docs/en/ingest-management/elastic-agent/elastic-agent-unprivileged-mode.asciidoc

```diff
@@ -0,0 +1,185 @@
+
+* <<unprivileged-running>>
+* <<unprivileged-command-behaviors>>
+* <<unprivileged-integrations>>
+* <<unprivileged-view-mode>>
+* <<unprivileged-change-mode>>
+
+[discrete]
+[[unprivileged-running]]
+== Run {agent} in `unprivileged` mode
+
+To run {agent} without administrative privileges you use exactly the same commands that you use for {agent} otherwise, with one exception. When you run the <<elastic-agent-install-command,`elastic-agent install`>> command, add the `--unprivileged` flag. For example:
+
+[source,shell]
+----
+elastic-agent install \
+  --url=https://cedd4e0e21e240b4s2bbbebdf1d6d52f.fleet.eu-west-1.aws.cld.elstc.co:443 \
+  --enrollment-token=NEFmVllaa0JLRXhKebVKVTR5TTI6N2JaVlJpSGpScmV0ZUVnZVlRUExFQQ== \
+  --unprivileged
+----
+
+IMPORTANT: On Linux systems, when you install {agent} using the `--unprivileged` flag, {agent} commands should not be run with `sudo`.
+Doing so may result in <<agent-sudo-error,an error>> due to the agent not having the required privileges.
+
+[discrete]
+[[unprivileged-command-behaviors]]
+== Agent and dashboard behaviors in unprivileged mode
+
+In addition to the <<unprivileged-integrations,integrations that are not available>> when {agent} is run in unpriviledged mode, certain data streams are also not available. The following tables show, for different operating systems, the impact when the agent does not have full administrative privileges. In most cases the limitations can be mediated by granting permissions for a user or group to the files indicated.
+
+.macOS
+[options,header]
+|===
+|Action |Behavior in unprivileged mode |Resolution
+
+|Run {agent} with the System integration
+|Log file error: `Unexpected file opening error: Failed opening /var/log/system.log: open /var/log/system.log: permission denied`.
+|Give read permission to the `elastic-agent` group for the `/var/log/system.log` file to fix this error.
+
+|Run {agent} with the System integration
+|On the `[Logs System] Syslog` dashboard, the `Syslog events by hostname`, `Syslog hostnames and processes` and `Syslog logs` visualizations are are missing data.
+|Give read permission to the `elastic-agent` group for the `/var/log/system.log` file to fix the missing visualizations.
+
+|Run {agent} with the System integration
+|On the `[Metrics System] Host overview` dashboard, only the processes ran by the `elastic-agent-user` user are shown in the CPU and memory usage lists.
+|Give read permission to the `elastic-agent` group for the `/var/log/system.log` file to fix the missing visualizations.
+
+|Run {agent} and access the {agent} dashboards
+|On the `[Elastic Agent] Agents info` dashboard, visualizations including `Most Active Agents` and `Integrations per Agent` are missing data.
+|Give read permission to the `elastic-agent` group for the `/var/log/system.log` file to fix the missing visualizations.
```

Comment:
I'm not sure this is the right resolution for this particular symptom.  @VihasMakwana would you mind confirming?

Reply:
Same `admin` group above applies.

Reply:
Changed as shown:
![Screenshot 2024-06-25 at 11 05 42 AM](https://github.com/elastic/ingest-docs/assets/41695641/35673a0c-1993-4935-8249-61ae07698bc3)